SPECS/bmake/bmake.spec

```diff
@@ -1,12 +1,13 @@
 Summary:       The NetBSD make(1) tool
 Name:          bmake
 Version:       20211221
-Release:       1%{?dist}
+Release:       2%{?dist}
 License:       BSD
 Vendor:        Microsoft Corporation
 Distribution:  Mariner
 URL:           https://ftp.netbsd.org/pub/NetBSD/misc/sjg/
 Source0:       %{url}/bmake-%{version}.tar.gz
+Patch0:        disable-time-tests.patch
```

Comment:
Please also add a comment with an explanation for this patch and a link to the bug, which forces us to apply it. This way in the future it's going to be easier to find it and check, if the issue is still unresolved.